app/views/listings/index.html.haml

```diff
@@ -0,0 +1,9 @@
+%h1 Listings#index
+- @listings.each do |listing|
+  #listing{:id=> "#{listing.id}" }
```

Comment:
please use the ActiveRecord method `dom_id` to set the id here

https://apidock.com/rails/ActionView/RecordIdentifier/dom_id

Reply:
Please use the new hash format